custom_components/mail_and_packages/const.py

```diff
@@ -188,6 +188,15 @@
     },
     "royal_packages": {},
     "royal_tracking": {"pattern": ["[A-Za-z]{2}[0-9]{9}GB"]},
+    "auspost_delivered": {
+        "email": ["noreply@notifications.auspost.com.au"],
+        "subject": ["Your shipment has been delivered"],
+    },
+    "auspost_delivering": {
+        "email": ["noreply@notifications.auspost.com.au"],
+        "subject": ["Your delivery is coming today"],
+    },
+    "auspost_packages": {},
```

Comment:
```suggestion
    "auspost_packages": {},
    "auspost_tracking": {},
```

Reply:
Based on the provided emails it looks like this would be the pattern to search for.

Reply:
I've added a few more combinations (I'm not sure if this is how the OR operator is used in this context)